### ISSUE TITLE
Tag 1.0.1 doesn't contain `dist` directory

### ISSUE THREAD

**Issue by Mogztter** [asciidoctor/asciidoctor-reveal.js#117]
Tag 1.0.1 doesn't contain `dist` directory: https://github.com/asciidoctor/asciidoctor-reveal.js/tree/v1.0.1

The following code is not working:
```
npm i --save asciidoctor-reveal.js@1.0.1
```

```javascript
require('asciidoctor-reveal.js');
```
```
Error: Cannot find module 'asciidoctor-reveal.js'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/guillaume/workspace/opensource/asciidoctor/test/test.js:5:1)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
```

**Comment by obilodeau** [comment]
I see. Not sure why it worked when I tested the 1.0.1 release...

I'll release 1.0.2 later today or this week. It'll contain #115 and misc doc fixes.

**Comment by obilodeau** [comment]
Fixed with the release of 1.0.2: 6401720

Packages have been pushed. I tested and it works. Re-open if necessary.
